docker/openmower_entrypoint.sh

```
@@ -22,10 +22,13 @@ if [[ "$MODE" == "osv1" ]]; then
     fi
     export ESC_TYPE=$OM_MOWER_ESC_TYPE
     export MOWER=$OM_MOWER
-fi
+    # source the hardware specific default environment (default wheel ticks, antenna position etc)
+    source "$(rospack find open_mower)/params/hardware_specific/$MOWER/default_environment.sh"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)